docs/generators/spring.md

```diff
@@ -113,6 +113,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |x-field-extra-annotation|List of custom annotations to be added to property|FIELD|null
 |x-spring-paginated|Add org.springframework.data.domain.Pageable to controller method. Can be used to handle page & size query parameters|OPERATION|false
 |x-version-param|Marker property that tells that this parameter would be used for endpoint versioning. Applicable for headers & query params. true/false|OPERATION_PARAMETER|null
+|x-pattern-message| Add this property whenever you need to customize the invalidation error message for the regex pattern of a variable|FIELD|
```

Comment:
we don't add this manually. Add this to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/VendorExtension.java + add it as supported extension to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java#L1241


once all this done you would need to re-generated docs using
`mvn clean && ./bin/utils/export_docs_generators.sh`